docs/docs/web-support.md

```diff
@@ -0,0 +1,23 @@
+---
+id: web-support
+title: Web Support
+sidebar_label: Web Support
+---
+
+Since
```

Comment:
I think we should mention Expo support too.

Reply:
I'm ok with that, could be another PR though

Reply:
or do you mean that we fire up web app using expo command?

Reply:
No, I meant that we should mention that Expo supports the experimental version of Rea2 in the native and web flavour